src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -2402,7 +2402,7 @@ def setter(vmss, external_identities=external_identities):
 
 # pylint: disable=too-many-locals, too-many-statements
 def create_vmss(cmd, vmss_name, resource_group_name, image=None,
-                disable_overprovision=False, instance_count=2,
+                disable_overprovision=None, instance_count=2,
```

Comment:
I would like to make sure that the default value change here will not cause breaking change, right?

Reply:
Default value change will cause breaking. May I know why you want to change the value here?

Reply:
Actually, what I mean is that the default value of `disable_overprovision` changes from `False` to `None` here

Reply:
It should be breaking as there is different from previous rest.

Reply:
Yes, but the default value has been changed in this PR

Reply:
As for flex mode, this property cannot be set. I will set it to False for uniform mode only

Reply:
see https://github.com/Azure/azure-cli/pull/18132/files#r661943085